test/quantization/test_quantize_fx.py

```diff
@@ -2962,31 +2965,46 @@ def get_available_classification_models(models):
         quantized_model_list = set(quantized_model_list) - no_pretrained_model
         # test eager and graph consistency
         model_list = quantized_model_list
-        # inception_v3 is not symbolically traceable: https://github.com/pytorch/pytorch/issues/48813
-        model_list = set(model_list) - {'inception_v3'}
-        # mobilenet: dropout error RuntimeError: "bernoulli_scalar_cpu_" not implemented for 'QUInt8'
-        # incpetion_v3: looks like there is some problem with AuxLogits
-        quantized_not_working = [('qat', 'inception_v3'),
-                                 ('static', 'inception_v3')]
-
-        fx_eager_not_matching = ['googlenet',  # because _transform_input is not quantized in eager
-                                 'mobilenet_v2']  # because relu6 is replaced as relu in mobilenetv2
+        model_list = set(model_list)
+        # mobilenet/inception_v3/googlenet qat is not working due to AdaptiveAveragePool qat
+        # we might observe the output of AdaptiveAveragePool in the future
+        # and re-enable the test
+        fx_eager_not_matching = [
+            ("mobilenet_v2", "qat"),
+            ("inception_v3", "qat"),
+            ("googlenet", "qat")
+        ]  # because relu6 is replaced as relu in mobilenetv2
 
         diff_of_quant = {}
         diff_from_eager = {}
         modes = ['static', 'qat']
         options = itertools.product(modes, model_list)
         for mode, name in options:
             pretrained = name in quantized_model_list  # load pretrained model to compare with quantized model
+            kwargs = {}
+            # turn off transform input for inception_v3 since
+            # it's not quantized in eager mode and in fx graph
+            # mode we can't skip quantizing a method right now
+            # (might be supported in the future)
+            if name in ["inception_v3", "googlenet"]:
+                kwargs["transform_input"] = False
+            eager_quantizable_model = None
             if name in quantized_model_list:
-                if (mode, name) in quantized_not_working:
-                    eager_quantizable_model = None
-                else:
-                    eager_quantizable_model = quantized_models.__dict__[name](pretrained=True, quantize=False).eval().float()
+                eager_quantizable_model = quantized_models.__dict__[name](pretrained=True, quantize=False, **kwargs).eval().float()
             # compare with eager mode quantized model when it is available
             pretrained = eager_quantizable_model is not None
-            model = models.__dict__[name](pretrained=pretrained).eval().float()
-            check_with_eager = name not in fx_eager_not_matching
+            model = models.__dict__[name](pretrained=pretrained, **kwargs).eval().float()
+            if name == "mobilenet_v2":
+                _replace_relu(model)
```

Comment:
Since you are replacing relu, why does mobilenetv2 not match?

Reply:
it matches in ptq static quantization, but failed in qat, because of AdaptiveAveragePool